types/rdfjs__fetch-lite/index.d.ts

```diff
@@ -5,10 +5,14 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import { DatasetCoreFactory, DatasetCore, Quad, Stream, BaseQuad } from '@rdfjs/types';
-import * as formats from '@rdfjs/formats-common';
+import { EventEmitter } from 'events';
+import { SinkMap } from '@rdfjs/sink-map';
 
 export interface FormatsInit extends RequestInit {
-    formats: typeof formats;
+    formats: {
+        parsers: SinkMap<EventEmitter, Stream>;
+        serializers: SinkMap<Stream, EventEmitter>;
+    };
```

Comment:
Could re-use the `Formats` type defined in types/rdfjs__express-handler/index.d.ts

Reply:
Did cross my mind. So export from formats-common?